tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -6,22 +6,23 @@ use crate::e2e::setup::constants::{
 use crate::e2e::setup::{Bin, Who};
 use crate::{run, run_as};
 
+/// this is the predictable bech32m address of the internal EthBridge account
 const ETH_BRIDGE_ADDRESS: &str = "atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew";
 
 /// # Examples
 ///
 /// ```
-/// let storage_key = storage_key("queue");
-/// assert_eq!(storage_key, "#atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew/queue");
+/// let storage_key = storage_key("ERC20/0x6B175474E89094C44Da98b954EedeAC495271d0F/supply");
+/// assert_eq!(storage_key, "#atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew/ERC20/0x6B175474E89094C44Da98b954EedeAC495271d0F/supply");
 /// ```
 fn storage_key(path: &str) -> String {
     format!("#{ETH_BRIDGE_ADDRESS}/{}", path)
 }
 
 #[test]
-fn everything() {
-    const LEDGER_STARTUP_TIMEOUT_SECONDS: u64 = 30;
-    const CLIENT_COMMAND_TIMEOUT_SECONDS: u64 = 30;
+fn unauthorized_tx_cannot_write_storage() {
+    const LEDGER_STARTUP_TIMEOUT_SECONDS: u64 = 60;
+    const CLIENT_COMMAND_TIMEOUT_SECONDS: u64 = 60;
```

Comment:
I *suspect* the flakes may have been to do with the timeouts being 30s (rather than the 40s we seem to use elsewhere), but clutching at straws a bit. We can try increasing this timeout and monitor for flakes going forward. A flaky error can happen at the first client assertion we make e.g.

```rust
anomac_tx.exp_string("Transaction accepted").unwrap();
```
Message:  called `Result::unwrap()` on an `Err` value: 
   0: Reached a timeout for expect type of command
       Needle: Transaction accepted
```

It's possible the ledger is shutting down for some reason before the client makes its call, or the client is making its call too quickly?

Separately from this PR, I wonder if this timeout value should be some tuneable configurable value, maybe from an environment variable. We use `Some(40)` seconds in most places but perhaps it should be a bit higher everywhere.